README.md

```diff
@@ -610,6 +606,18 @@ CLI updates the cache on every call to a `terra resource` command. So, if you
 are working in a shared workspace, you can run `terra resource list` (for
 example) to pick up any changes that your collaborators have made.
 
+##### list-tree
```

Comment:
I would delete 609-620. We don't do these for any other `resource` commands. `list-tree` is pretty self-explanatory.